source/features/comment-shortcuts.tsx

```diff
@@ -0,0 +1,79 @@
+import select from 'select-dom';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+const shortcutClass = new Map<string, string>([
+	['e', '.rgh-edit-comment, [aria-label="Edit comment"]'],
+	['d', '[aria-label="Delete comment"]'],
+	['h', '[aria-label="Hide comment"]'],
+	['j', ''],
+	['k', '']
+]);
+
+function commentHash(comment: HTMLElement): string {
+	// What do I call this var??
+	const reviewComment = comment.closest('.js-comment[id^="pullrequestreview-"]');
+	const {hash} = reviewComment ? reviewComment.querySelector<HTMLAnchorElement>('a.js-timestamp')! : comment.querySelector<HTMLAnchorElement>('a.js-timestamp')!;
+
+	return reviewComment ? hash + '-body-html' : hash;
+}
+
+function runShortcuts(event: delegate.Event<KeyboardEvent>): void {
+	if (!shortcutClass.has(event.key) || event.target instanceof HTMLTextAreaElement || event.target instanceof HTMLInputElement) {
+		return;
+	}
+
+	event.preventDefault();
+
+	if (['j', 'k'].includes(event.key)) {
```

Comment:
3/4
```suggestion
	if (['j', 'k'].includes(event.key)) {
		event.preventDefault();
```

Reply:
We need to prevent default. Otherwise when doing <kbd>e</kbd> for edit the `e` will be placed in the comment